tests/SixLabors.Fonts.Tests/Tables/AdvancedTypographic/Gsub/GSubTableTests.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using System.Linq;
+using System.Numerics;
+using Xunit;
+
+namespace SixLabors.Fonts.Tests.Tables.AdvancedTypographic.Gsub
+{
+    public class GSubTableTests
+    {
+        // TODO: add all other arabic letters
+        [Fact]
+        public void RenderArabicCharacters_WithIsolatedForm_Works()
+        {
+            // arrange
+            Font arabicFont = new FontCollection().Add(TestFonts.ArabicFontFile).CreateFont(8);
+            string testStr = "ب";
+            var rendererTtf = new ColorGlyphRenderer();
+            bool applyKerning = true;
+            Vector2[] expectedControlPoints = new[]
```

Comment:
for this test would it not make more sense to figure out what the expected glyph index should be and validate its the one tryign to be rendered rather than looking at control points? 

You can access the rendered glyph index using the `GlyphKeys` collection on  `ColorGlyphRenderer`

Reply:
yeah thats much better, glad you mentioned it before i did tests for all the other characters 😄